docs/en/sql-reference/statements/create/table.md

```diff
@@ -405,6 +405,10 @@ These codecs are designed to make compression more effective by using specific f
 
 `Gorilla(bytes_size)` — Calculates XOR between current and previous floating point value and writes it in compact binary form. The smaller the difference between consecutive values is, i.e. the slower the values of the series changes, the better the compression rate. Implements the algorithm used in Gorilla TSDB, extending it to support 64-bit types. Possible `bytes_size` values: 1, 2, 4, 8, the default value is `sizeof(type)` if equal to 1, 2, 4, or 8. In all other cases, it’s 1. For additional information, see section 4.1 in [Gorilla: A Fast, Scalable, In-Memory Time Series Database](https://doi.org/10.14778/2824032.2824078).
 
+#### GCD
```

Comment:
L. 403 would be a slightly better position for the GCD docs (Delta and DoubleDelta are equally "preparation-only" codecs, whereas Gorilla, FPC and T64 are "true" codecs)